tests/error-handling.test.ts

```
@@ -85,7 +85,7 @@ test.describe('Error Handling and Edge Cases', () => {
     await page.close();
   });
 
-  test('should handle network timeouts appropriately', async (_, testInfo) => {
+  test('should handle network timeouts appropriately', async ({}, testInfo) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)